api/v1/verticadb_webhook.go

```diff
@@ -818,6 +821,26 @@ func (v *VerticaDB) hasValidClientServerTLSMode(allErrs field.ErrorList) field.E
 	return allErrs
 }
 
+// hasTLSSecretsSetForRevive checks whether the TLS secrets are set for the revive init policy
+// when TLS is enabled
+func (v *VerticaDB) hasTLSSecretsSetForRevive(allErrs field.ErrorList) field.ErrorList {
+	if vmeta.UseTLSAuth(v.Annotations) && v.Spec.InitPolicy == CommunalInitPolicyRevive {
+		if v.GetHTTPSNMATLSSecret() == "" {
```

Comment:
Libo's list did not include revive. I already added it to my PR with good unit test coverage. I suggest we remove it here. Moreover we also need to include v.Spec.NMATLSsecret at this line, which I already did in my PR.